test/functional/admin/role_appointments_controller_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class Admin::RoleAppointmentsControllerTest < ActionController::TestCase
   setup do
-    login_as :writer
+    login_as_preview_design_system_user(:gds_editor)
```

Comment:
```
  login_as_preview_design_system_user(:writer)
```

Ideally, we should keep the same level of permissions on the user so it's testing the exact same stuff.